### DISCUSSION TITLE
Show the actual like count when pressing on a video's description

### DISCUSSION THREAD

**Discussion by Ascyt** [ReVanced/revanced-discussions#133] [Question]
When you press on the description you can currently see the exact view count (for example 1.237.103 instead of 1.2M) and I think the same should work for the like button. 

![Screenshot_20220329-210030](https://user-images.githubusercontent.com/81428303/160687742-d2d14221-888d-4655-9491-e317358f975d.jpg)

(here you can see how it isn't showing the exact like count)

**Comment by MSOB7YY** [comment]
i agree with the views, but like count would be adding more distraction imo

**Comment by Ascyt** [comment]
> i agree with the views, but like count would be adding more distraction imo

How? There is more than enough space to show it

**Comment by Sadaweya0** [comment]
I want this a lot too.

**Comment by MSOB7YY** [comment]
> > i agree with the views, but like count would be adding more distraction imo
> 
> How? There is more than enough space to show it

i mean, when u want to check the likes count, it's a distraction to see the full count
something like 92k would be a lot better than 92,486

**Reply by RobinRMC** [reply]
> > > i agree with the views, but like count would be adding more distraction imo
> > 
> > 
> > How? There is more than enough space to show it
> 
> i mean, when u want to check the likes count, it's a distraction to see the full count something like 92k would be a lot better than 92,486

It's only visible when you open the video description. I agree that this would be a useful feature to have, as YouTube can't round numbers properly.

**Comment by stackingflow** [comment]
I don't see any point of showing the like count up to that specificity

70k like or 70,122 - later one is too much info to process and would make the layout complicated and cluttered.

**Comment by Ascyt** [comment]
> > > i agree with the views, but like count would be adding more distraction imo
> > 
> > 
> > How? There is more than enough space to show it
> 
> i mean, when u want to check the likes count, it's a distraction to see the full count something like 92k would be a lot better than 92,486

Then what's the point of showing the full view count?

**Comment by Ascyt** [comment]
> I don't see any point of showing the like count up to that specificity
> 
> 70k like or 70,122 - later one is too much info to process and would make the layout complicated and cluttered.

Then what is the point of showing the exact view count
